charts/vineyard-operator/README.md

```diff
@@ -6,39 +6,94 @@ A helm chart for [vineyard operator][3], which manages all relavant components a
 
 ## Install
 
-Vineyard operator has been integrated with [Helm](https://helm.sh/). Deploy it as follows:
+Vineyard operator has been integrated with [Helm](https://helm.sh/). Add the vineyard repository to your Helm client:
+
+```shell
+
+$ helm repo add vineyard https://vineyard.oss-ap-southeast-1.aliyuncs.com/charts/
+$ helm repo update
+
+```
+
+Refer to the [helm repo](https://helm.sh/docs/helm/helm_repo/) for more repo information.
+
+The webhook is enabled by default, please make sure you have the [Cert-Manager](https://cert-manager.io/docs/installation/) installed, then deploy it in the `default` namespace as follows:
+
+```shell
+
+$ helm install vineyard-operator vineyard/vineyard-operator
 
-```console
-helm repo add vineyard https://vineyard.oss-ap-southeast-1.aliyuncs.com/charts/
-helm install vineyard-operator vineyard/vineyard-operator
 ```
 
-Install vineyardd as follows.
+If you want to deploy it in a specific namespace, you can use the `--namespace` option:
 
-.. code:: shell
+```shell
+
+$ helm install vineyard-operator vineyard/vineyard-operator \
+      --namespace vineyard-system
+   
+```
 
-   curl https://raw.githubusercontent.com/v6d-io/v6d/main/k8s/test/e2e/vineyardd.yaml | kubectl apply -f -
+If you want to set the value of the chart, you can use the `--set` option:
+
+```shell
+
+$ helm install vineyard-operator vineyard/vineyard-operator \
+      --set image.tag=v0.10.1
+
+```
+
+Refer to the [helm install](https://helm.sh/docs/helm/helm_install/) for more command information.
+
+Install `vineyardd` as follows.
+
+```shell
+
+$ cat <<EOF | kubectl apply -f -
+apiVersion: k8s.v6d.io/v1alpha1
+kind: Vineyardd
+metadata:
+  name: vineyardd-sample
+  # don't use default namespace
+  namespace: vineyard-system
+spec:
+  image: ghcr.io/v6d-io/v6d/vineyardd:alpine-latest
+  replicas: 2
+  imagePullPolicy: IfNotPresent
+  syncCRDs: true
+  enableMetrics: false
+  etcd:
+    replicas: 3
+  service:
+    type: ClusterIP
+    port: 9600
```

Comment:
Omit default value please.

Please refer to the revised example in `vineyard-operator.rst`.